velox/functions/prestosql/aggregates/MapAggAggregate.cpp

```diff
@@ -23,8 +25,11 @@ namespace {
 template <typename K>
 class MapAggAggregate : public MapAggregateBase<K> {
  public:
-  explicit MapAggAggregate(TypePtr resultType)
-      : MapAggregateBase<K>(std::move(resultType)) {}
+  explicit MapAggAggregate(
+      TypePtr resultType,
+      const bool throwOnNestedNulls = false)
```

Comment:
```
bool throwOnNestedNulls = false
```
Drop const qualifier for scaler input type.

Reply:
Fixed